### DISCUSSION TITLE
Favicon used by web UI should be distinct from favicon used for documentation tabs on home-assistant.io

### DISCUSSION THREAD

**Discussion by domesticatedviking** [home-assistant/frontend#19048] [Other feature requests]
It would be very helpful if the favicon used by the web UI was visually distinct from the favicon used by home-assistant.io.

When working on integrations I am constantly losing track of the open UI tab.   If the UI used a favicon with a different color or shape, this would not be as much of an issue.

To demonstrate why this matters -- let's play a game called "which of these tabs has my local server's UI running."

![image](https://github.com/home-assistant/frontend/assets/6627466/90cee585-3cee-4675-b24e-4e33849a19dd)

I'm tired of playing this game.  Please and thanks.

**Comment by ogland** [comment]
Dogpile!
